.travis.yml

```diff
@@ -1,13 +1,18 @@
 language: go
-script: "./test.sh"
 go:
-    - 1.5
-    - 1.6
-    - 1.7
-    - tip
+  - 1.5.4
+  - 1.6.3
```

Comment:
That's not how gimme works.  You have to specify the version explicitly.  See the [latest build](https://travis-ci.org/pelletier/go-toml/jobs/154093148#L127) and notice that it builds with go1.6.  Compare it to [this PR build](https://travis-ci.org/pelletier/go-toml/jobs/153802919#L135).